en/ui-system/components/engine/ui-material.md

```diff
@@ -0,0 +1,47 @@
+# UI Custom Material
+We supports UI custom material in sprite component since v1.2, the user interface is as follows:
+
+![](ui-material/UIMaterial.png)
+
+Its usage is no different from other materials, but because the SpriteComponent has some function based on the UI-built-in materials, there are some points to note:
+
+- When the number of custom materials is set to 0 or empty, the default material will be used. Please refer to [SpriteComponent](../editor/sprite.md)
```

Comment:
Please refer to [SpriteComponent](../editor/sprite.md)
-->
Please refer to the [SpriteComponent](../editor/sprite.md) documentation.

Reply:
this is unresolved. referenced in the latest review